### DISCUSSION TITLE
Any way to export .pdf files with and without notes to oneself? (similar to Beamer notes but not for presentations)

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#10746] [Q&A]
### Description

Hi all,

I give presentations not using slides but handouts. Is there a "note" function, similar to that in Beamer but for ordinary pdf files? 

I mean, I would like to create a handout, and I would like to include my personal notes. But I do not want to have two separate files. I would like to have a single file, with the options of including or not including my notes in the exported pdf file. 

Can I do this? Thank you.

**Comment by mcanouil** [comment]
Please check the documentation: https://quarto.org/docs/presentations/revealjs/#speaker-notes

Or other presentation formats including Beamer.
